portfolio/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,58 @@
+@font-face {
+    font-family: selima;
+    src: url("../fonts/Selima\ .otf");
+}
+
+/* Header Section with navbar and image */
+#banner {
```

Comment:
Optional: It's generally best practice to avoid using IDs as selectors within CSS (i.e. use `header` instead of `#banner`). Marking optional since there's some interesting debate over this topic.

If you're curious: https://www.quora.com/Why-is-it-considered-bad-form-to-use-an-ID-selector-in-a-CSS-stylesheet